### ISSUE TITLE
Warnings on forms with out-of-the-box settings

### ISSUE THREAD

**Issue by rosswintle** [WordImpress/Give#29]
Installed Give on a local dev site where I have verbose error reporting. I get warnings on forms like this:

```
Warning: Illegal string offset 'currency_position' in /Users/rosswintle/Dropbox/Oikos/Plugins/wordpress-plugin-dev/wp-content/plugins/Give/includes/forms/template.php on line 221

Warning: Illegal string offset 'currency' in /Users/rosswintle/Dropbox/Oikos/Plugins/wordpress-plugin-dev/wp-content/plugins/Give/includes/forms/template.php on line 222
```

**Comment by DevinWalker** [comment]
Thanks for the report

**Comment by DevinWalker** [comment]
This is fixed now as of #40
